<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="scalesLabel">
       <property name="text">
        <string>Scales</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="scaleLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="backgroundGroupBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string>Backgrounds</string>
       </property>
       <layout class="QVBoxLayout" name="backgroundLayout"/>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="backgroundLabel">
       <property name="text">
        <string>Default background</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="backgroundComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="providersLabel">
       <property name="text">
        <string>Providers</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="coordinatesCheckBox">
       <property name="text">
        <string>coordinates</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="nominatimCheckBox">
       <property name="text">
        <string>nominatim</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
